src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -242,4 +243,33 @@ public Object getData(String string) {
 		}
 		return list;
 	}
+
+	public List<String> getPKStrings() {
+		ArrayList<String> list = new ArrayList<>();
+
+		String pkString = "db:" + table.getDatabase().getName() + "/"
+				        + "tbl:" + table.getName() + "/";
```

Comment:
this is counter-intuitive. You expect getPKStrings to get you PK strings only, but sure.